### PR TITLE
Improve document in adapter.md

### PR DESCRIPTION
**Pull request by LiShuMing**

### CODE REVIEW

File: site/_docs/adapter.md [hunk 1]
```diff
@@ -107,14 +107,14 @@ as implemented by Avatica's
...
-  jdbc:calcite:schemaType=JDBC; schema.jdbcUser=SCOTT; schema.jdbcPassword=TIGER; schema.jdbcUrl=jdbc:hsqldb:res:foodmart
+  `jdbc:calcite:schemaType=JDBC; schema.jdbcUser=SCOTT; schema.jdbcPassword=TIGER; schema.jdbcUrl=jdbc:hsqldb:res:foodmart`
 
 creates a connection with a schema mapped via the JDBC schema adapter to the foodmart database.
 
 Similarly, you can connect to a single schema based on a user-defined schema adapter.
 For example,
 
-  jdbc:calcite:schemaFactory=org.apache.calcite.adapter.cassandra.CassandraSchemaFactory; schema.host=localhost; schema.keyspace=twissandra
+  `jdbc:calcite:schemaFactory=org.apache.calcite.adapter.cassandra.CassandraSchemaFactory; schema.host=localhost; schema.keyspace=twissandra`
```
Comment (danny0405) [inline site/_docs/adapter.md]: Can we merge these doc changes into one PR ?

Comment (LiShuMing) [reply site/_docs/adapter.md]: Maybe I reset the codes here?

**Comment by julianhyde** [comment]
We don’t use ‘[MINOR]’.

**Comment by LiShuMing** [comment]
Sorry， I will edit it .

**Pull request closed without merging**